modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManagerImpl.java

```diff
@@ -2686,7 +2685,7 @@ private void finishUpdate(GridCacheContext cctx, CacheDataRow newRow, @Nullable
 
             GridCacheQueryManager qryMgr = cctx.queries();
 
-            if (qryMgr.enabled())
+            if (qryMgr.enabled() || cctx.kernalContext().indexing().enabled())
```

Comment:
Is it possible to have a single flag for this?
Seems, it make no sense having Indexing enabled, but qryMgr disabled.

Reply:
> no sense having Indexing enabled, but qryMgr disabled.

Hi. It was a workaround, because for some reasons this condition was required at beginning of development. Currently I don't see in code any reasons to have this condition. So I'll remove it and check that everything is fine.

Reply:
Removed.